credit-conveyor/Deal/src/main/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+server.port= 9090
```

Comment:
better to use incremental numbering in ports. Ex: 8080 for one service, 8081 for another, 8082 for another etc